CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 1.  [#3256](https://github.com/influxdata/chronograf/pull/3256): Reduce font sizes in dashboards for increased space efficiency
 1.  [#3320](https://github.com/influxdata/chronograf/pull/3320): Add overlay animation to Template Variables Manager
 1.  [#3245](https://github.com/influxdata/chronograf/pull/3245): Display 'no results' on cells without results
+1.  [#3353](https://github.com/influxdata/chronograf/pull/3353): YAxisLabels in Dashboard Graph Builder not showing until graph is redrawn
```

Comment:
could you write this in a way that an end user would understand what behavior they can now expect, that doesn't contain anything that sounds 'code-ish' (like `YAxisLabels`), such as `Always show graph's y-axis label while using dashboard graph builder`?